Foundation/include/Poco/NumberFormatter.h

```diff
@@ -573,17 +737,33 @@ inline std::string NumberFormatter::format0(unsigned int value, int width)
 
 
 inline std::string NumberFormatter::formatHex(unsigned value, bool prefix)
+{
+	return formatHex(value, prefix ? Options::OPT_PREFIX : Options::OPT_NONE);
+}
+
+
+inline std::string NumberFormatter::formatHex(unsigned value, Options options)
 {
 	std::string result;
-	intToStr(value, 0x10, result, prefix);
+	const bool prefix { (options & Options::OPT_PREFIX) != Options::OPT_NONE};
+	const bool lowercase { (options & Options::OPT_LOWERCASE) != Options::OPT_NONE};
```

Comment:
Just asking about styling. Do Podo now use {} initializing? Still if can be used which I like is it like this that bracket is seperated from variable name? "lowercase{" vs "lowecase {" also I think at least last } should also include space "Options::OPT_NONE}" vs "Options::OPT_NONE }"

Reply:
what we should have for cases like this is style checking with clang-tidy or some similar tool. lots of time gets wasted on style violations that could be resolved before code is pushed

Reply:
Definitly! That is basically most time saving thing what can be done. In companies but also in open source. Style checking is done with clang-format. Needs little bit hacking to get it work with everything but it is worth a time.

Reply:
I did not see a rule against brace initialization in coding style (or I missed it). Will change to parentheses.

Reply:
Resolved in new commit.